examples/textConnect/contents/setting.html

```diff
@@ -4,210 +4,218 @@
  *
  * Licensed under the MIT License
 -->
-<div>  
-  <div class="block">
-    <label class="kintoneplugin-label">
-        <span id ="container_label">結合する項目</span>
-    </label><br >
-    <div class="kintoneplugin-desc">結合する項目を選択してください。（最大5つまで）</div>
-    
-    <div class="kintoneplugin-row" id="row1">
-        <span class="num">①</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select1" id= "select1">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select2" id= "select2">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select3" id= "select3">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select4" id= "select4">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select5" id= "select5">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
+<div id="plugin-container">
+
+</div>
```

Comment:
This div probably won't be needed due to the removal of line 95 of config.js

Reply:
Updated the codes. Please see the latest commit made on 8/17.